game.js

```diff
@@ -25,22 +25,33 @@ heroImage.onload = function () {
   heroReady = true;
 };
 heroImage.src = "images/hero.png";
+
 // Load the monster image
-var monsterReady = false;
-var monsterImage = new Image();
-monsterImage.onload = function () {
+var monster1Ready = false;
+var monster1Image = new Image();
+monster1Image.onload = function () {
+  // show the monster image
+  monster1Ready = true;
+};
+monster1Image.src = "images/monster.png";
+
+var monster2Ready = false;
+var monster2Image = new Image();
+monster2Image.onload = function () {
   // show the monster image
-  monsterReady = true;
+  monster2Ready = true;
 };
-monsterImage.src = "images/monster.png";
+monster2Image.src = "images/monster.png";
+var monstersCaught = 2;
 // Create the game objects
 var hero = {
   speed: 256, // movement speed of hero in pixels per second
   height: 36,
   width: 31
 };
+
 var monster = {};
-var monstersCaught = 0;
+var monstersCaught = 1;
```

Comment:
Why are you changing this? This is simply keeping track of how many monsters you have caught so far